src/main/java/org/cascadebot/cascadebot/music/CascadePlayer.java

```diff
@@ -251,13 +251,18 @@ public void loadPlaylist(String name, Member sender, PlaylistType scope, BiConsu
     }
 
     private void loadLoadedPlaylist(Playlist playlist, long reqUser, Consumer<List<AudioTrack>> loadedConsumer) {
+        List<AudioTrack> tracks = new ArrayList<>();
         for (String url : playlist.getTracks()) {
             loadLink(url, reqUser, noMatch -> {
                 playlist.removeTrack(url);
             }, exception -> {
                 playlist.removeTrack(url);
-            }, loadedConsumer);
+            }, tracks::addAll);
+        }
+        while (tracks.size() < playlist.getTracks().size()) {
```

Comment:
I really don't like this, find a better way to do it

Reply:
be my guest

Reply:
This also will infinite loop if any of them fail 🤔

Reply:
We remove them from the playlist if they fail.